braid/braid.c

```diff
@@ -1499,3 +1525,48 @@ braid_Rand(void)
    return (unsigned int) (_braid_rand_next/65536) % braid_RAND_MAX;
 }
 
+/*--------------------------------------------------------------------------
+ *--------------------------------------------------------------------------*/
+
+braid_Int
+braid_SetRichardsonEstimation(braid_Core core,
+                              braid_Int  est_error,
+                              braid_Int  richardson,
+                              braid_Int  local_order)
+{
+   
+   braid_Real  *CWts        = _braid_CoreElt(core, CWts);
+   braid_Real   CWt_default = _braid_CoreElt(core, CWt_default);
+   braid_Int    ml          = _braid_CoreElt(core, max_levels);
+   braid_Int j;
+
+   /* Compatability checks */
+   if ( local_order < 2 && ( est_error || richardson ) )
+   {
+      _braid_printf(" Local Order of Time Integrator must be > 1 \n\n" );
+      abort();
+   }
+
+   if(CWt_default != 1.0)
+   {
+      printf("\nWeighted relaxation and Richardson extrapolation are incompatible.  Ignoring Richardson options.\n\n");
```

Comment:
@jbschroder Looks like we have a `_braid_printf` that we should be using?

Reply:
@rfalgout Good point!  I'll change that to _braid_printf here in a minute.  So many coding changes...and it must have slipped my mind :-)

Reply:
FYI, note that there are other places that use `printf` .  I can change them if you want.  Just let me know.